packages/preview/colorful-boxes/1.3.1/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "colorful-boxes"
+version = "1.3.1"
+repository = "https://github.com/lkoehl/typst-boxes"
+entrypoint = "lib.typ"
+authors = ["Lukas Köhl <https://github.com/lkoehl>"]
+license = "MIT"
+description = "Predefined colorful boxes"
+exclude = [ "/examples/*"]
```

Comment:
Hey, while you're updating the package, may I suggest adding the `categories` key to the manifest? This will make the package show up in the respective category on https://typst.app/universe. You can find [a list of the categores here](https://github.com/typst/packages/blob/main/CATEGORIES.md).

Reply:
Great idea! Added the category